core/templates/dev/head/css/oppia.css

```diff
@@ -1818,13 +1818,15 @@ pre.oppia-pre-wrapped-text {
   font-size: 12px;
 }
 .tooltip.bottom {
```

Comment:
All the changes made here, affect every tooltip class. That should not be the case. Instead add a new class for the tooltip being fixed and then add the new styling to that.

Reply:
(Ditto for the other two changes as well)